README.md

```diff
@@ -31,7 +31,7 @@ Check font styles. Use [Playfair Display](https://fonts.google.com/specimen/Play
 11. `git push origin develop` - to send you code for PR.
 12. Create a Pull Request (PR) from your branch `develop` to branch `master` of original repo.
 13. Replace `<your_account>` with your Github username in the
-  [DEMO LINK](https://<your_account>.github.io/Museum_2/).
+  [DEMO LINK](https://DmytroPAvlichenko/.github.io/Museum_2/).
```

Comment:
```suggestion
  [DEMO LINK](https://DmytroPAvlichenko.github.io/Museum_2/).
```

Reply:
BTW, link in PR description is still not working: https://dmytropavlichenko.github.io/Museum_2/